systers_portal/static/js/libs/jquery-ui-timepicker-0.3.3.js

```diff
@@ -0,0 +1,1496 @@
+/*
```

Comment:
I see you added jquery-ui as a minified version. I am wondering if it would make sense to have a minified version of jquery-ui-timepicker as well. You can keep this copy locally for development.